pkg/controller/serviceaccount/tokens_controller_test.go

```diff
@@ -483,9 +483,6 @@ func TestTokenCreation(t *testing.T) {
 				secrets.Add(tc.AddedSecret)
 				controller.queueSecretSync(tc.AddedSecret)
 			}
-			if tc.AddedSecretLocal != nil {
```

Comment:
this is the only use of AddedSecretLocal ... dropping this means a testcase using that is probably not testing what it thinks it is